src/services/adherence.service.ts

```diff
@@ -142,8 +142,39 @@ async function getAdherenceAlerts(
   return result.data
 }
 
+async function markAdherenceAlertsAsRead(
+  studyId: string,
+  alertIds: string[],
+  token: string
+): Promise<any> {
+  const endpoint = constants.endpoints.adherenceAlertsMarkRead.replace(':studyId', studyId)
```

Comment:
here I would use adherenceAlerts endpoint constant replace studyId as you are doing and then concatenate '/unread' or '/read' depending on the param